org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/codelens/ICodeLensResolver.java

```diff
@@ -16,10 +16,10 @@
 /**
  * @author Sven Efftinge - Initial contribution and API
  */
-@SuppressWarnings("all")
 public interface ICodeLensResolver {
-  /**
-   * Resolve the given code lens.
-   */
-  CodeLens resolveCodeLens(final Document document, final XtextResource resource, final CodeLens codeLens, final CancelIndicator indicator);
-}
+	
+	/**
+	 * Resolve the given code lens.
+	 */
+	CodeLens resolveCodeLens(Document document, XtextResource resource, CodeLens codeLens, CancelIndicator indicator);
+}
```

Comment:
Not sure how Xtext feels about this but I very much like a newline at the end of the file